addons/web/static/src/views/view.js

```diff
@@ -10,14 +10,37 @@ import { useActionLinks } from "@web/views/helpers/view_hook";
 
 const viewRegistry = registry.category("views");
 
-const { Component } = owl;
+const { Component, hooks } = owl;
+const { useSubEnv } = hooks;
+
+export const makeActionContext = (...values) => {
```

Comment:
this should maybe be moved to the action_service (like props, there is a common basis for both client and window actions, and then some keys only available for window actinos)